src/main/java/com/spotify/annoy/ANNIndex.java

```diff
@@ -8,53 +8,69 @@
 import java.util.*;
 
 /**
- * Read-only Approximate Nearest Neighbor Index which queries databases created by annoy.
+ * Read-only Approximate Nearest Neighbor Index which queries
+ * databases created by annoy.
  */
-public class ANNIndex implements AnnoyIndex {
```

Comment:
@edwardnewett added this AnnoyIndex thing... I take it you guys don't need it anymore?